tests/AopCodeTest.php

```
@@ -20,15 +21,16 @@
 
 class AopCodeTest extends TestCase
 {
-    /** @var AopCode */
-    private $codeGen;
+    private const TMP_DIR = __DIR__ . '/tmp';
```

PHPStan (2.1.32)

Constant Ray\Aop\AopCodeTest::TMP_DIR is unused.
See: https://phpstan.org/developing-extensions/always-used-class-constants

(classConstant.unused)